nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc

```diff
@@ -12,6 +12,31 @@ These release notes track the development of the {secondary-scheduler-operator-f
 
 For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
 
+[id="secondary-scheduler-operator-release-notes-1.4.1_{context}"]
+== Release notes for {secondary-scheduler-operator-full} 1.4.1
+
+Issued: 9 July 2025
+
+The following advisory is available for the {secondary-scheduler-operator-full} 1.4.1:
+
+* link:https://access.redhat.com/errata/RH??-2025:TODO[RH??-2025:TODO]
+// TODO: Update advisory links
+
+[id="secondary-scheduler-1.4.1-new-features_{context}"]
+=== New features and enhancements
+
+* This release of the {secondary-scheduler-operator} updates the Kubernetes version to 1.32.
+
+[id="secondary-scheduler-1.4.1-bug-fixes_{context}"]
+=== Bug fixes
+
+* This release of the {secondary-scheduler-operator} addresses several Common Vulnerabilities and Exposures (CVEs).
+
+[id="secondary-scheduler-operator-1.4.1-known-issues_{context}"]
+=== Known issues
+
+* Currently, you cannot deploy additional resources, such as config maps, CRDs, or RBAC policies through the {secondary-scheduler-operator}. Any resources other than roles and role bindings that are required by your custom secondary scheduler must be applied externally. (link:https://issues.redhat.com/browse/WRKLDS-645[*WRKLDS-645*])
```

Comment:
On core OCP release notes, we removed bold formatting from bug numbers as they were visually overpowering on DRC.

Reply:
Also the same known issue needs to exist for 1.4.1 and 1.4.0?